### PR TITLE
dev-libs/hyperscan: Version bump to 5.0.0

### PR DESCRIPTION
**Pull request by croessner**

### CODE REVIEW

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: Please update your calendar.

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+SRC_URI="https://github.com/01org/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://01.org/hyperscan"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_ssse3 static-libs"
+
+DEPEND="dev-util/ragel
+	=dev-lang/python-2*
```
Comment (mgorny) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: Depending on `dev-lang/python` directly is unacceptable. Use appropriate eclass instead (read docs on https://wiki.gentoo.org/wiki/Project:Python).

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	dev-libs/boost
```
Comment (mgorny) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: `:=`

File: dev-libs/hyperscan/hyperscan-5.0.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	net-libs/libpcap"
+
+REQUIRED_USE="cpu_flags_x86_ssse3"
```
Comment (mgorny) [inline dev-libs/hyperscan/hyperscan-5.0.0.ebuild]: Looks like I won't be able to test it :-(.

**Comment by mgorny** [comment]
Ping.

**Comment by croessner** [comment]
I have created a new PR. https://github.com/gentoo/gentoo/pull/9862

**Pull request closed without merging**